css/styles.css

```diff
@@ -71,154 +71,177 @@ body {
 
 /*=== media queries ===*/
 @media screen and (min-width: 401px) {
-  .ideas {
-    display: grid;
-    grid-template-columns: 20% 40% 40%;
-    grid-template-rows: 25% 25% 25% 25%;
-  }
-  .search-view {
-    grid-area: 1/ 2/ 1/ 1;
-  }
-  .idea-form {
-    grid-area: 1/ 5/ 3/ 2;
-  }
-  .idea-cards {
-    grid-area: 3/ 5/ 3/ 2;
-    display: grid;
-    grid-template-columns: 1fr 1fr 1fr;
-    grid-template-rows: 1fr 1fr 1fr 21fr;
-  }
-  .nav-view-container{
-    height: 70em;
-    /* display: flex; */
-    flex-direction: column;
-    padding: 4%;
-  }
-  .show-starred-button{
-    border: none;
-    background-color: #5356a4;
-    color: #ffffff;
-    font-size: 1.4em;
-    padding: 3%;
-  }
-  .nav-view-header{
-    align-self: center;
-    color: #ffffff;
-    font-size: 1.3em;
-  }
-  .nav-view-subheader{
-    color: #a9aad2;
-    padding: 3% 0% 7% 0%;
-  }
-
-  .nav-menu {
-    display: none;
+    .ideas {
```

Comment:
Good work, Orlando! I'm curious about the use of em instead of percentage with some of these fixes. My understanding is that ems should be used for font sizing, but I may be wrong. Let's discuss it during our project time today.

Reply:
I’m also noticing that the mobile layout seems to no longer fit the 400x parameter